imports/ui/components/dashboard/matches_section.jsx

```diff
@@ -13,19 +15,19 @@ const MatchesSection = () => {
               </Nav.Link>
             </Nav.Item>
             <Nav.Item>
-              <Nav.Link eventKey='pastMatches'>Past Matches</Nav.Link>
+              <Nav.Link eventKey='otherSubmissions'>Other Submissions</Nav.Link>
```

Comment:
Maybe instead of `submissions`, we should keep our vocabulary consistent and call it `entries`?

Reply:
Sounds good!